tests/test_config.py

```diff
@@ -717,12 +711,10 @@ async def test_samesite_valid_config():
         )
 
 
+# TODO: Check if we need to create new applications for each subtest
```

Comment:
TODOs

Reply:
Addressed a few in 5fdd04c.
Logged an issue to pick up TODOs added for test parametrization: #570.